### ISSUE TITLE
[BUG] macOS crash with `wstring_convert::from_bytes` exception

### ISSUE THREAD

**Issue by pgmccann** [aristocratos/btop#277]
**Read the README.md and search for similar issues before posting a bug report!**

Any bug that can be solved by just reading the [prerequisites](https://github.com/aristocratos/btop#prerequisites) section of the README will likely be ignored.

**Describe the bug**

btop crashes with the error message:

```
ERROR: Exception in runner thread -> Proc:: -> wstring_convert::from_bytes
```

**To Reproduce**

Run `btop`
Leave running for a few minutes

**Expected behavior**

btop not to crash

**Info (please complete the following information):**
 - btop++ version: 1.2.3
 - Binary: release (via Homebrew)
 - (If compiled) Compiler and version:
 - Architecture: x86_64
 - Platform: macOS
 - (Linux) Kernel: `uname -r`
 - (OSX/FreeBSD) Os release version: 12.2.1
 - Terminal used: iTerm2 3.5.0beta4
 - Font used: FiraCode Nerd Font Mono

**Additional context**

contents of `~/.config/btop/btop.log`

```
2022/01/19 (10:02:54) | ===> btop++ v.1.1.4
2022/01/19 (10:02:54) | ERROR: Stall in Runner thread, restarting!
2022/01/21 (21:45:09) | ERROR: failed getting network interfaces

2022/01/27 (17:04:25) | ===> btop++ v.1.2.0
2022/01/27 (17:04:25) | ERROR: Stall in Runner thread, restarting!
2022/01/27 (17:19:23) | ERROR: Stall in Runner thread, restarting!

2022/02/07 (17:52:14) | ===> btop++ v.1.2.0
2022/02/07 (17:52:14) | ERROR: failed getting network interfaces

2022/02/23 (15:43:59) | ===> btop++ v.1.2.3
2022/02/23 (15:43:59) | ERROR: Exception in runner thread -> Proc:: -> wstring_convert::from_bytes

2022/02/24 (10:52:07) | ===> btop++ v.1.2.3
2022/02/24 (10:52:07) | ERROR: Exception in runner thread -> Proc:: -> wstring_convert::from_bytes

2022/02/24 (11:19:44) | ===> btop++ v.1.2.3
2022/02/24 (11:19:44) | ERROR: Exception in runner thread -> Proc:: -> wstring_convert::from_bytes
```

**GDB Backtrace**

If btop++ is crashing at start the following steps could be helpful:

(Extra helpful if compiled with `make OPTFLAGS="-O0 -g"`)

1. run (linux): `gdb btop` (macos): `lldb btop`

2. `r` to run, wait for crash and press enter if prompted, CTRL+L to clear screen if needed.

3. (gdb): `thread apply all bt` (lldb): `bt all` to get backtrace for all threads

4. Copy and paste the backtrace here:

```
(lldb) target create "btop"
Current executable set to 'btop' (x86_64).
(lldb) r
Process 91008 launched: '/usr/local/bin/btop' (x86_64)
ERROR: Failed to get size of terminal!
Process 91008 exited with status = 1 (0x00000001)
(lldb) bt all
error: invalid thread
```

(This does not appear to be the same issue, but may be of interest)

**Comment by aristocratos** [comment]
@pgmccann 
This is fixed in commit 7b3d996
I will likely push a new release in the coming days if you don't want to compile from source.

**Comment by aristocratos** [comment]
Fixed in v1.2.4

**Comment by pgmccann** [comment]
Still seeing this in 1.2.4

**Comment by aristocratos** [comment]
@pgmccann 
Should be fixed in latest commits

**Comment by aristocratos** [comment]
Fixed in v1.2.5
